### PR TITLE
Add service objectives page

### PR DESCRIPTION
**Pull request by choldgraf**
This adds a rough service level objectives page to our documentation. The goal is to lay down some basic guidelines for what we strive for in the hub service. It also makes explicit our commitment to protecting the team's time over weekends and holiday periods (and thus is related to https://github.com/2i2c-org/meta/issues/257)

It is not meant to be comprehensive, and is intentionally not extremely detailed. It makes explicit that this is not a legal promise, but our team's *objectives*.

### CODE REVIEW

File: about/service-objectives.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+We may revisit this in the future depending on the feedback we get from other communities!
+:::
+
+## High availability
+
+The infrastructure that 2i2c runs should be available to its communities 24/7, and with minimal human intervention needed to maintain this level of performance.
+We invest in continous development to improve the resiliency and efficiency of the infrastructure that we run, following best-practices in service design and engineering in the cloud.
+
+:::{admonition} To be refined...
+It is a known anti-pattern to define an ambiguous SLO like "24/7".
```
Comment (damianavila) [inline about/service-objectives.md:23]: This is a really good point I was going to raise, nice to see it already written.

File: about/service-objectives.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+There is an inherent tension between doing things quickly (which generally requires using extra resources to encourage speed) and cost efficiency (because you pay for those extra resources).
+This is particularly relevant during **scaling events**.
+These are moments when the infrastructure has enough usage that it must grow the cloud resources available to handle the new load.
+
+2i2c strives to build infrastructure that strikes a balance that depends on the particular use-case.
+If infrastructure requires steady, but semi-random usage, we should prioritize cost efficiency.
+If infrastructure will have known spikes of activity at the same time, we may temporarily favor speed over cost by asking for extra resources from the cloud provider.
+
+:::{note}
+If your community requires a change in the infrastructure that occurs over a weekend, we will generally try to do this on the Friday beforehand, rather than over the weekend, even if this means it will cost marginally more in cloud infrastructure.
```
Comment (damianavila) [inline about/service-objectives.md:39]: Great note, again!

**Review by damianavila — APPROVED** [review]
I really like the content on this PR!

**Review by colliand — APPROVED** [review]

File: about/service-objectives.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+# Service Level Objectives and Principles
+
+This page describes the **Service Level Objectives** of 2i2c's infrastructure services[^slos].
```
Comment (sgibson91) [inline about/service-objectives.md:3]:
```suggestion
This page describes the **Service Level Objectives** (SLOs) of 2i2c's infrastructure services[^slos].
```

**Review by sgibson91 — APPROVED** [review]

**Comment by choldgraf** [comment]
I'd love to get @yuvipanda's 👍 on this one before we merge, since he's got the most experience with SLO-style workflows. Anything major we should change here Yuvi?

**Comment by choldgraf** [comment]
I'll plan to merge this in the next day unless somebody objects, thanks all for the feedback.

**Pull request merged**